<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="14" hostname="DESKTOP-H3O127Q" name="regressionTests.RegressionCases" tests="15" failures="1" timestamp="06 Aug 2019 05:36:11 GMT" time="571.190" errors="0">
  <testcase name="Verify_Offer_Rename_Is_Successfull" time="571.190" classname="regressionTests.RegressionCases">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at regressionTests.RegressionCases.Verify_Offer_Rename_Is_Successfull(RegressionCases.java:114)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Verify_Offer_Rename_Is_Successfull -->
  <testcase name="Verify_Deleted_Offer_Is_Not_Present_In_CSV" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_Deleted_Offer_Is_Not_Present_In_CSV -->
  <testcase name="Verify_NewProperty_Added_Is_Available_in_DB" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_NewProperty_Added_Is_Available_in_DB -->
  <testcase name="Verify_Discard_Version_Is_Successful" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_Discard_Version_Is_Successful -->
  <testcase name="Verify_Creating_NewMicrositeTemp_ByAdding_Existing_Attribute" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_Creating_NewMicrositeTemp_ByAdding_Existing_Attribute -->
  <testcase name="Verify_Existing_Attribute_Can_be_Added_In_OCCTemplate" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_Existing_Attribute_Can_be_Added_In_OCCTemplate -->
  <testcase name="Verify_NewProperty_Added_Is_Available_in_PegaClass" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_NewProperty_Added_Is_Available_in_PegaClass -->
  <testcase name="Verify_DownloadedCSV_For_Existing_Version" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_DownloadedCSV_For_Existing_Version -->
  <testcase name="Verify_RealTimeEvent_API_Response_For_NewEvent" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_RealTimeEvent_API_Response_For_NewEvent -->
  <testcase name="Verify_AdaptiveModel_For_Negative_Response" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_AdaptiveModel_For_Negative_Response -->
  <testcase name="Verify_Rollback_Is_Working_As_Expected" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_Rollback_Is_Working_As_Expected -->
  <testcase name="Verify_DownloadedCSV_For_New_version" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_DownloadedCSV_For_New_version -->
  <testcase name="Verify_RealtimeSpine_Api" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_RealtimeSpine_Api -->
  <testcase name="Verify_IH_Deleted_For_All" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_IH_Deleted_For_All -->
  <testcase name="Verify_NewEmailTemp_ByCloning" classname="regressionTests.RegressionCases">
    <skipped/>
  </testcase> <!-- Verify_NewEmailTemp_ByCloning -->
</testsuite> <!-- regressionTests.RegressionCases -->
